source/user-manual/reference/centralized-configuration.rst

```diff
@@ -120,26 +120,52 @@ Options
 +-------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | **name**    | Allows assignment of the block to one particular agent.                                                                                                           |
 +             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-|             | Allowed values                                        | Any agent name                                                                                            |
+|             | Allowed values                                        | Any regular expression that matches the agent name .                                                      |
 +-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
 | **os**      | Allows assignment of the block to an operating system.                                                                                                            |
 +             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-|             | Allowed values                                        | Any OS family                                                                                             |
+|             | Allowed values                                        | Any regular expression that matches the agent OS information.                                             |
 +-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
 | **profile** | Allows assignment of a profile name to a block. Any agent configured to use the defined :ref:`profile <reference_ossec_client_config_profile>` may use the block. |
```

Comment:
```suggestion
| **profile** | Assigns the block to agents with specific profiles as defined in:ref:`reference_ossec_client_config_profile`.                                                     |
```

Reply:
I changed a little bit this proposal because it was not showing properly the reference after compilation.